### ISSUE TITLE
[Release Docs Issue] Outdated Checksum for Install_NDI_SDK_v5_Linux.tar.gz in Install Instructions

### ISSUE THREAD

**Issue by orithena** [obs-ndi/obs-ndi#882]
The checksum given in the Linux Installation Instructions for Release 4.11.1 currently does not match the downloaded NDI SDK package. As a consequence, the given install script fails:
```
Install_NDI_SDK_v5_Linux.tar.gz: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match
```
Seems like NDI did release a new version while reusing the file name?

Installing the `libndi5_5.5.3-1_amd64.deb` package instead did work for me, though. So, it's only a case of "instructions unclear".

**Comment by lazzarello** [comment]
I changed this

```
# LIBNDI_INSTALLER_SHA256="00d0bedc2c72736d82883fc0fd6bc1a544e7958c7e46db79f326633d44e15153"
LIBNDI_INSTALLER_SHA256="c16db57b467d06cce1455393df744f152623b48ae53d76aa6ac4b3b5f0e3bde8"
```

**Comment by Nikoh77** [comment]
Hello guys, just to inform that the installation instructions are still wrong...

**Comment by jenswes** [comment]
Hello,
today i've tried this, but checksum / download changed again

```
wget  https://downloads.ndi.tv/SDK/NDI_SDK_Linux/Install_NDI_SDK_v5_Linux.tar.gz
sha256sum Install_NDI_SDK_v5_Linux.tar.gz 
7e5c54693d6aee6b6f1d6d49f48d4effd7281abd216d9ff601be2d55af12f7f5  Install_NDI_SDK_v5_Linux.tar.gz

```

```
# LIBNDI_INSTALLER_SHA256="00d0bedc2c72736d82883fc0fd6bc1a544e7958c7e46db79f326633d44e15153"
# LIBNDI_INSTALLER_SHA256="c16db57b467d06cce1455393df744f152623b48ae53d76aa6ac4b3b5f0e3bde8"
LIBNDI_INSTALLER_SHA256="7e5c54693d6aee6b6f1d6d49f48d4effd7281abd216d9ff601be2d55af12f7f5"

```
the above is working.

thanks for your work

**Comment by paulpv** [comment]
Thanks all!
I have noticed this too for a long time.
Since that file name stays the same but the contents keep changing, I am thinking of just taking that checksum test out entirely.

**Comment by paulpv** [comment]
This will be removed in 4.12.0

**Comment by paulpv** [comment]
Fixed in #919
